.circleci/real_config.yml

```diff
@@ -345,15 +345,36 @@ commands:
         type: string
       python-version:
         type: string
-        default: "3.8.16"
+        default: "3.8.18"
     steps:
+      - run:
+          name: Write cache key
+          command: |
+            echo <<parameters.executor>> > /tmp/cachefile
+            pip freeze --all >> /tmp/cachefile
```

Comment:
Should this be `freeze --all` or perhaps just `--local`?

Reply:
The line was copied since this was moved up

Curious why the suggestion for ```--local```? I would have though our cache key would want to include as much difference as possible?